configuration/configure-automatic-upgrades.adoc

```diff
@@ -0,0 +1,39 @@
+[id="configure-automatic-upgrades"]
+= Configuring automatic upgrades for secured clusters
+include::modules/common-attributes.adoc[]
+:context: configure-automatic-upgrades
+
+toc::[]
+
+[role="_abstract"]
+You can automate the upgrade process for each secured cluster and view the upgrade status from the RHACS portal.
+
+Automatic upgrades make it easier to stay up-to-date by automating the manual task of upgrading each secured cluster.
+
+With automatic upgrades, after you upgrade Central, the Sensor, Collector, and Compliance services in all secured clusters automatically upgrade to the latest version.
```

Comment:
Should there be a comma after "and Compliance"?

Reply:
It was confusing, I rewrote it as:
```
With automatic upgrades, after you upgrade Central;  Sensor, Collector, and 
Compliance services in all secured clusters, automatically upgrade to the latest version.
```

Reply:
Thanks for clarifying, I definitely didn't interpret it that way. 

Just one last thing - I don't think you should have the comma that you added if it's meaning what I think it is. So if it's saying that, if you have automatic upgrades, then after you upgrade central, then those other 3 components in all secured clusters will automatically upgrade to the latest version? 

I don't want to hold up the PR over it, so I'm going to go ahead and merge this, but I'd consider removing the comma in a future PR.